components/Header/Header.jsx

```diff
@@ -16,61 +15,65 @@ const Header = ({
   right,
   headerTextColor,
 }) => {
-  const {setLoggedIn} = useContext(GlobalDataContext);
-
-  const handleLogout = async key => {
-    await removeItem(key);
-    setLoggedIn(false);
-    Alert.alert('Logout successful');
+  const [isModalVisible, setModalVisible] = useState(false);
+  const handleLogout = () => {
```

Comment:
const handleLogout = () => setModalVisible(true)

Reply:
I will fix this.